client/src/pages/pot-instance/styles.module.css

```diff
@@ -6,6 +6,13 @@
   left: 47vw;
 }
 
+.potImgContainer {
+  position: absolute;
+  top: 25vh;
+  left: 20vw;
+  right: 20vw;
+}
+
```

Comment:
This is styling for the pot images. It is using absolute to make sure it stays in the center of the page. 

I noticed a bug where the pot images overlay ontop of the detailed timers making them unclickable. Do we want to use z-index or explore a different option?

Reply:
Do you wanna put the pot under the list. We could add some transparency or spacing so it will still peek through.
To fix the unclickable issue you could move the `potImgContainer` element further up in the parent component, probably above `PotView`

Reply:
![image](https://user-images.githubusercontent.com/107276941/191134879-6311f0a7-0d02-4c1b-98ae-69a6582a719b.png)

I noticed that `potImgContainer` is above `PotView` currently (it set on the main/first component). I tried moving it down, but it doesn't fix this issue either? I updated the z-index for both the `completedTimer` + `foodTimer` component and it is now behaving what I expect it to be (see image).

My questions on this:
1) Originally, the `potImgcontainer` is on PotInstance and the `timer` components are even further below it. Shouldn't the timer components render _after_ the `potImageContainer` making it be on top? 

My understanding is that things that are rendered later should be on top (like when you click on the `+` to add Ingredients, it re-renders that component to go on top of everything else)?

But this isint the case with `potImgcontainer` and the `timer` components.

2) Now that each `timer` component has z-index = 1, why do newly rendered components still override this value (like when the `addIngredient` component gets rendered? Shouldn't the default value be _lower_ than the new `timer` components?

Reply:
1. Yep you are right about the rendering order. 
When you click on the `+`, there is a another component that gets generated independent of the button,( this is placed that the top layer of the dom), with a high z-index so that its above everything, like what you just did.

With the unclickable issues I think the pot image and buttons were at a similar level so that button was actually blocked by the hidden parts of the div. 
I realized that when you moved it up the div width shrank so the buttons were clickable).

2. Not sure what you mean by this.